src/utils.py

```diff
@@ -102,17 +111,18 @@ def __init__(self,
         self.pred_list = []
         self.pred_proba_list = []
         self.label_list = []
-        self.metrics_dict = {'accuracy': [], 'f1score': [], 'precision': [], 'recall': [], 'auc': []}
+        self.metrics_dict = {'accuracy': [], 'f1score': [],
+                             'precision': [], 'recall': [], 'auc': []}
```

Comment:
Can you change your max line length in the code editor? it create a lot of diffs and it's hard to follow..

Reply:
Set the value to?

Reply:
120 characters is default seems to be sufficient since most of us use IntelliJ's Pycharm